forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-gateway/forgerock-openbanking-uk-aspsp-rs-gateway-server/src/test/java/com/forgerock/openbanking/aspsp/rs/api/payment/verifier/DetachedJwsVerifierTest.java

```diff
@@ -162,6 +164,21 @@ public void shouldFailToVerifyB64HeaderGivenVersion3_1_4AndB64HeaderIsPresent()
         assertThat(exception).hasMessage("Invalid detached signature " + detachedJws + ". Reason: b64 claim header must not be present");
     }
 
+    @Test
+    public void shouldVerifyB64HeaderGivenVersion3_1_4AndB64HeaderIsMissing() throws Exception {
```

Comment:
Is this a duplicate of `shouldVerifyDetachedJwsGivenVersion3_1_4AndB64HeaderIsMissing()`?

Reply:
No. One is testing that verification works when < 3.1.4 and b64 is false, the other tests that >= 3.1.4 will verify when b64 is not present.

Reply:
I think you might be comparing it to a different test?